seesawlabs/seesawlabs.github.io
# RockstarVR page

Added the RockstarVR WebGL build to the page at `/rockstarVR/`.

To test:
1. Pull down this branch `git checkout --track origin/rockstar`.
2. Run `jekyll serve` from project directory
3. Navigate to [localhost:4000/rockstarVR/](http://localhost:4000/rockstarVR).
4. Make sure the loading screen works, the styling looks okay, and that the RockstarVR experience is working and smooth.
